src/view/screens/Search/Search.tsx

```diff
@@ -850,6 +864,7 @@ function AutocompleteResults({
     </>
   )
 }
+AutocompleteResults = React.memo(AutocompleteResults)
```

Comment:
@gaearon why you dont just wrap the component with memo?
maybe I'm missing some context

Reply:
would need a Babel plugin to keep function names readable without adding an extra indentation level due to writing them twice